spaces/S000083/properties/P000145.md

```diff
@@ -0,0 +1,11 @@
+---
+space: S000083
+property: P000145
+value: true
+---
+
+Suppose $\mathcal U$ is an open cover of $X$.  The subspace $Y=(\mathbb R\setminus\{0\})\cup\{0_1\}$ is open in $X$ and homeomorphic to $\mathbb R$, hence strongly paracompact.  Intersecting every element of $\mathcal U$ with $Y$ gives an open cover of $Y$, and, since {S25|P145}, we can choose a star finite open refinement $\mathcal V$ of that cover of $Y$. 
```

Comment:
```suggestion
Suppose $\mathcal U$ is an open cover of $X$.  The subspace $Y=(\mathbb R\setminus\{0\})\cup\{0_1\}$ is open in $X$ and homeomorphic to $\mathbb R$.  Intersecting every element of $\mathcal U$ with $Y$ gives an open cover of $Y$, and, since {S25|P145}, we can choose a star finite open refinement $\mathcal V$ of that cover of $Y$. 
```
remove redundancy (it's the same as `{S25|P145}` in the next sentence)